content/docs/command-reference/remote/modify.md

```diff
@@ -75,6 +75,22 @@ The following config options are available for all remote types:
   $ dvc remote modify localremote url /home/user/dvcstore
   ```
 
+- `jobs` - number of workers started at execution of the following commands:
+
+  - push
+  - pull
+  - status
+  - gc
+  - fetch
+
+  Accepts positive integers. The default value is `4 * cpu_count()`. For SSH
+  remotes, the default is `4`. Using more jobs may improve the overall transfer
+  speed.
```

Comment:
same here, not sure transfer applies well to `gc`

Reply:
also `status`

Reply:
This is useful for most of the commands, so I don't think this is worth clarifying more here.

Reply:
Using "connection speed" here as well for now.

Reply:
`connection speed` has a very precise meaning though, and `--jobs` doesn't change it. `speedup the operation` sounds good. For pull/push - etc transfer is fine.

Reply:
"speed up the operation" it is, in gc and status. Note that in remote modify I removed that sentence. Applied in 89dd8269 (see https://github.com/iterative/dvc.org/pull/1945#pullrequestreview-531868327).